server/src/main/java/org/candlepin/model/AsyncJobStatus.java

```diff
@@ -28,32 +31,34 @@
 import java.util.HashMap;
 import java.util.Map;
 
+import javax.persistence.CollectionTable;
 import javax.persistence.Column;
-import javax.persistence.Convert;
 import javax.persistence.Converter;
+import javax.persistence.ElementCollection;
 import javax.persistence.Entity;
 import javax.persistence.FetchType;
 import javax.persistence.GeneratedValue;
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
 import javax.persistence.ManyToOne;
+import javax.persistence.MapKeyColumn;
 import javax.persistence.Table;
 import javax.persistence.Version;
 import javax.validation.constraints.NotNull;
-import javax.xml.bind.annotation.XmlAccessType;
-import javax.xml.bind.annotation.XmlAccessorType;
-import javax.xml.bind.annotation.XmlRootElement;
 
 
 
 /**
- * Represents the current status for a long-running job.
+ * Represents the job status for an async job. This class does not include extended information,
+ * such as the job result. The AsyncJobStatusEx class can be used in cases where the extended
+ * information is needed.
  */
-@XmlRootElement
-@XmlAccessorType(XmlAccessType.PROPERTY)
 @Entity
 @Table(name = AsyncJobStatus.DB_TABLE)
-public class AsyncJobStatus extends AbstractHibernateObject implements JobExecutionContext {
+public class AsyncJobStatus extends AbstractHibernateObject {
+
+    /** Basic ObjectMapper to use for serializing job results as we receive them */
+    private static final ObjectMapper MAPPER = ObjectMapperFactory.getObjectMapper();
```

Comment:
Now that we use an objectmapper excplicitly here, to serialize result objects, we should remove the old infrastructure we used to do that with the `@Convert` annotation: JobDataJsonConverter / AbstractJsonConverter.java

Reply:
You can do it that way if you want, or leave it as a todo?